### PR TITLE
make Touchable component platform independant

### PR DESCRIPTION
**Pull request by l3satwik**

### CODE REVIEW

File: src/components/shared/Touchable.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import {
+  TouchableWithoutFeedbackProps,
+  Platform,
+  TouchableNativeFeedback,
+  TouchableHighlight
+} from "react-native";
+
+const Touchable: React.FunctionComponent<TouchableWithoutFeedbackProps> = ({
```
Comment (ritz078) [inline src/components/shared/Touchable.tsx]: with this, I won't be able to change ripple tint color as it won't be an acceptable prop.

Comment (l3satwik) [reply src/components/shared/Touchable.tsx]: `TouchableProps = TouchableNativeFeedbackProps | TouchableHighlightProps`?

Comment (ritz078) [reply src/components/shared/Touchable.tsx]: yes

File: src/components/shared/Touchable.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+type TouchableProps = TouchableNativeFeedbackProps | TouchableHighlightProps;
+
+const Touchable: React.FunctionComponent<TouchableProps> = ({
+  children,
+  ...props
+}) => {
+  if (Platform.OS === "android") {
+    return (
+      <TouchableNativeFeedback {...props}>{children}</TouchableNativeFeedback>
```
Comment (azizhk) [inline src/components/shared/Touchable.tsx]: How are you able to pass TouchableHighlightProps to TouchableNativeFeedback
and vice versa.
This should be throwing an error.

**Pull request merged**